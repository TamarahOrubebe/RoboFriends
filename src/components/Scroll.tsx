import React, { ReactNode } from 'react';

interface IProps{
    children: ReactNode
}

const Scroll = (props: IProps) => {


    return (
        <div style={{overflow: "scroll", height: "800px", border: "2px solid black"}}>
            {props.children}
        </div>
    )
}


export default Scroll;