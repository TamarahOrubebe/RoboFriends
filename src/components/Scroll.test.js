import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

it("renders a Scroll component", () => {
	
	expect(shallow(<Scroll />)).toMatchSnapshot();
});
