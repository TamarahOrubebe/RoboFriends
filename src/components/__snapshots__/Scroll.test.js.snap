// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Scroll component 1`] = `
<div
  style={
    Object {
      "border": "2px solid black",
      "height": "800px",
      "overflow": "scroll",
    }
  }
/>
`;
